tests/endtoend/test_linux_consumption.py

```diff
@@ -1,4 +1,4 @@
-from unittest import TestCase, skip
+from unittest import TestCase
```

Comment:
Can you copy and put the copyright comment (from any other file) to this file as well?